tutorial/csharp.async/README.md

```diff
@@ -0,0 +1,8 @@
+# Tutorial Frameword 4.5+ Clent Async
+
+## Setp
+1. thrift -gen csharp:async shared.thrift
+2. thrift -r -gen csharp:async tutorial.thrift
+3. Put it in Tutorial folder
+4. Generated && Run
```

Comment:
generate, not generated

Reply:
I have a general problem wth the workflow. IMHO even if we copy the files manually, we should keep the folder structure. Hence, I'd suggest to have the user copy the entire gen-csharp folder as is, not single files.